src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs

```diff
@@ -27,6 +27,8 @@ internal static class Vector128
         public static Vector128<ulong> AsUInt64<T>(this Vector128<T> vector) where T : struct => throw new PlatformNotSupportedException();
         public static T GetElement<T>(this Vector128<T> vector, int index) where T : struct => throw new PlatformNotSupportedException();
         public static T ToScalar<T>(this Vector64<T> vector) where T : struct => throw new PlatformNotSupportedException();
+        public static unsafe Vector128<ulong> CreateScalar(ulong value) => throw new PlatformNotSupportedException();
+        public static T ToScalar<T>(this Vector128<T> vector) where T : struct => throw new PlatformNotSupportedException();
```

Comment:
I don't see the usage of these APIs inside `ASCIIUtility.cs`. Any reason why this is added?

Reply:
I needed them for my next PR, which I'll have up soon.

Reply:
My opinion is that you add to this PR only the shims that you need.